sentinel-extension/sentinel-annotation-aspectj/src/main/java/com/alibaba/csp/sentinel/annotation/aspectj/SentinelResourceAspect.java

```diff
@@ -59,12 +59,33 @@ public Object invokeResourceWithSentinel(ProceedingJoinPoint pjp) throws Throwab
         } catch (BlockException ex) {
             return handleBlockException(pjp, annotation, ex);
         } catch (Throwable ex) {
-            Tracer.trace(ex);
+            if (isTrackedException(ex, annotation.exceptionsToTrace())) {
```

Comment:
Maybe we can move the logic to `AbstractSentinelAspectSupport` class to support extension for sub-classes. E.g.:

```java
protected void traceException(Throwable ex, SentinelResource annotation)
```